clients/src/test/java/org/apache/kafka/common/protocol/types/ProtocolSerializationTest.java

```diff
@@ -278,4 +278,53 @@ public void testStructEquals() {
         assertNotEquals(emptyStruct1, mostlyEmptyStruct);
         assertNotEquals(mostlyEmptyStruct, emptyStruct1);
     }
+
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6340 instead. Thanks!